kairos_face/enroll.py

```diff
@@ -1,15 +1,15 @@
 import base64
 
 import requests
-
+import numpy as np
 from kairos_face import exceptions
 from kairos_face import settings
 from kairos_face.utils import validate_file_and_url_presence, validate_settings
 
 _enroll_base_url = settings.base_url + 'enroll'
 
 
-def enroll_face(subject_id, gallery_name, url=None, file=None, additional_arguments={}):
+def enroll_face(subject_id, gallery_name, url=None, file=None, imgframe=None multiple_faces=False, additional_arguments={}):
```

Comment:
I'd change the new parameter name to `image_base64_contents` or something like this to make its intent clearer.